src/style/global-style.js

```diff
@@ -34,6 +34,33 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+a {
+  border-left: 1px solid #dddddd;
+}
```

Comment:
We can't add a border to all `a` elements. For example, this breaks the "About" page: https://test.studio.opencast.org/2022-04-08_13-18-12-pr921-2115380323/about

Why did you add this and remove the borderLeft thing from `header.js`?